cypress/e2e/tests/pages/global-settings/settings-p2.spec.ts

```diff
@@ -0,0 +1,419 @@
+import { SettingsPagePo } from '@/cypress/e2e/po/pages/global-settings/settings.po';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import BurgerMenuPo from '@/cypress/e2e/po/side-bars/burger-side-menu.po';
+import ClusterManagerCreateRke2CustomPagePo from '@/cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create-rke2-custom.po';
+import AccountPagePo from '@/cypress/e2e/po/pages/account-api-keys.po';
+import ClusterManagerListPagePo from '@/cypress/e2e/po/pages/cluster-manager/cluster-manager-list.po';
+import { settings } from '@/cypress/e2e/blueprints/global_settings/settings-data';
+
+const settingsPage = new SettingsPagePo('local');
+const homePage = new HomePagePo();
+const accountPage = new AccountPagePo();
+const clusterList = new ClusterManagerListPagePo('local');
+const burgerMenu = new BurgerMenuPo();
+
+describe('Settings', { testIsolation: 'off' }, () => {
+  before(() => {
+    cy.login();
+    HomePagePo.goTo();
+  });
+
+  it.skip('can update server-url', { tags: ['@globalSettings', '@adminUser'] }, () => {
+    // Note: this test fails sometimes due to https://github.com/rancher/dashboard/issues/9690
+    // skipping this test until issue is resolved
+
+    // Update setting
+    SettingsPagePo.navTo();
+
+    // Get original value and store it via aliasing
+    settingsPage.settingsValue('server-url').then((el: any) => {
+      const originalValue = el.text();
+
+      cy.wrap(originalValue).as('originalValue');
+    });
+
+    settingsPage.editSettingsByLabel('server-url');
+
+    const settingsEdit = settingsPage.editSettings('local', 'server-url');
+
+    settingsEdit.waitForPage();
+    settingsEdit.title().contains('Setting: server-url').should('be.visible');
+    settingsEdit.settingsInput().set(settings['server-url'].new);
+    settingsEdit.saveAndWait('server-url');
+    settingsPage.waitForPage();
+    settingsPage.settingsValue('server-url').contains(settings['server-url'].new);
+
+    // Check Account and API Keys page
+    accountPage.goTo();
```

Comment:
are the goTo's here needed?

Reply:
Updated to `navTo`s. I was able to make it so that there is only one `goTo` left (besides the one in the `beforeAll` hook) for navigating to the cluster list page (`clusterList.goTo()`). That one needs to be there bc when updating it to `clusterList.navTo()` it causes issues with subsequent tests